### DISCUSSION TITLE
To implement whisper API support to generate text instead of running it natively

### DISCUSSION THREAD

**Discussion by vikesh001** [SubtitleEdit/subtitleedit#6905] [Ideas]
Dear development team,

I would like to request the addition of Whisper API support to your application for text generation. By integrating OpenAI's language models via the Whisper API, developers can generate high-quality text on demand and offload computationally intensive tasks to OpenAI's servers.

With Whisper API support, developers would be able to improve the performance and quality of their applications while reducing costs associated with scaling infrastructure. This approach also offers the added benefits of improved scalability, flexibility, and reliability, making it an excellent choice for any developer seeking high-quality text generation capabilities.

To use Whisper API, developers must configure their application to communicate with the API endpoint, providing inputs such as desired length and context. The API will then generate text based on these inputs, returning it to the application for use in various contexts, including chatbots, content generation, or creative writing.

By implementing Whisper API support, your application could streamline development processes and free up valuable resources to focus on other aspects of the application. I believe this feature would be a valuable addition to your already impressive set of features and provide significant benefits to developers utilizing your application.

Thank you for considering my feature request.

**Comment by niksedk** [comment]
A PR is welcome.

Running native for free is probably what most people want.
